helm/starboard-exporter/Chart.yaml

```diff
@@ -1,10 +1,12 @@
 apiVersion: v1
 appVersion: [[ .AppVersion ]]
 name: starboard-exporter
-description: A Helm chart for starboard-exporter
+description: A Helm chart for starboard-exporter, which exposes Prometheus metrics from Aqua VulnerabilityReport and other custom resources.
 engine: gotpl
 home: https://github.com/giantswarm/starboard-exporter
+sources:
+- https://github.com/aquasecurity/starboard
```

Comment:
Small thing, and I appreciate the intent, but I think this is not technically the intended usage for `sources`. From Helm:
```
sources:
  - A list of URLs to source code for this project (optional)
```

Reply:
I think this comes from another PR and shouldn't be here :) Removing.